sunscreen_runtime/src/linked.rs

```diff
@@ -89,6 +89,7 @@ fn new_single_party_with_shared_generators(
         if left_side {
             g[index] = *gen;
             left_side = false;
+            // TODO this overflows if shared arguments are provided with no constraints
```

Comment:
Similar to above, something that needs to be tackled when we shore up the shared commitments, but unrelated to this PR.

Reply:
As in the ZKP takes in a shared input but does nothing with them?

Reply:
Yea exactly